README.md

```diff
@@ -19,18 +19,18 @@ To run the **HR Management** Web App, ensure your system meets the following req
 Follow the steps below to setup the project locally:
 
 * clone the repository: 
-git clone https://github.com/harry-olabs/avi-hub.git
+git clone ```https://github.com/harry-olabs/avi-hub.git```
+
 * Navigate to the project directory: 
 cd hr-management-web-app
 
 ## Install dependencies using npm 
 
-* *npm* install or *yarn* install
+* ```npm install```
```

Comment:
Please read the GitHub documentation again to see to format code blocks:

Take a look at [Quoting Code](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code)